content/02.main-text.md

```diff
@@ -69,3 +69,30 @@ The randomization procedure was shared with the authors in advance (pre-register
 Given the same author contributions, it always produced the same ordered author list.
 We annotated the author list to indicate that author order was partly randomized and emphasize that the order did not indicate one author contributed more than another from the same category.
 `TODO: In Discussion, present alternative author ordering strategies and literature on contribution in collaborative projects`
+
+### Discussion
+
+Many others have embraced open science principles and  piloted open approaches toward drug discovery [@doi:10.1371/journal.pmed.1002276], data management [@doi:10.1371/journal.pbio.1001195; @doi:10.1038/sdata.2016.18; @doi:10.1038/nbt.3516], and manuscript review [@doi:10.12688/f1000research.12037.1].
```

Comment:
Under citations for "open approaches toward drug discovery" would love to cite Project Rephetio https://doi.org/10.1101/087619

Reply:
Yes, definitely!  I added it.

These examples were quite arbitrary so please let me know what other ideas you have for open science projects to highlight.